extensions/amp-base-carousel/0.1/auto-advance.js

```diff
@@ -83,8 +83,14 @@ export class AutoAdvance {
     this.createDebouncedAdvance_(this.autoAdvanceInterval_);
     this.scrollContainer_.addEventListener(
         'scroll', () => this.handleScroll_(), true);
-    this.scrollContainer_.addEventListener(
-        'touchstart', () => this.handleTouchStart_(), true);
+    listen(
+        this.scrollContainer_,
+        'touchstart',
+        () => this.handleTouchStart_(),
+        {
```

Comment:
nit: to make this less ugly after `prettier`, maybe set the option obj as a one-liner.

`{capture: true, passive: true}`.

this is a mostly stable API so it's unlikely more params will be added. in any case `prettier` would fix it automatically if it surpasses line length.

Reply:
I can't get it to behave. I end up with:

```javascript
    listen(
      this.scrollContainer_,
      'touchstart',
      () => this.handleTouchStart_(),
      {capture: true, passive: true}
    );
    listen(this.scrollContainer_, 'wheel', () => this.handleWheel_(), {
      capture: true,
      passive: true,
    });
```

If I try to do:

```javascript
    listen(
      this.scrollContainer_,
      'wheel', 
      () => this.handleWheel_(),
     {capture: true, passive: true}
    );
```

it forces the options argument to multiple lines for the second `listen`, since it sees it can fit the start of the object literal on the previous line.